es-design-system/pages/atoms/typography.vue

```diff
@@ -74,15 +75,34 @@
                 </b-table>
             </b-col>
         </b-row>
+        <b-row class="my-5 border-top pt-5">
+            <b-col>
+                <h2>
+                    Font Size Utility Classes
```

Comment:
Visual hierarchy is lost on those using screen-readers i.e. utility classes don't convey the semantics of the markup. Ensuring proper hierarchy in headings will also lend to easier navigation & potentially easier parsing by search-engine-crawlers.

Could we add a small note here, maybe something like ["Don't rely only on the appearance of the font (color, shape, font variation, placement, etc.) to convey meaning."](https://a11y-style-guide.com/style-guide/section-general.html#kssref-general-typography)?

Reply:
Points taken I will add some documentation. The primary use cases for these classes are not headings or elements meant to convey hierarchy.

Reply:
Totally, they're super useful for building components. Documentation should be enough.